.github/workflows/release.yml

```diff
@@ -0,0 +1,37 @@
+name: Release
+
+on:
+  release:
+    types:
+      - published
```

Comment:
Just a note, currently the workflow is run when a new `release` is published.
https://docs.github.com/en/actions/reference/events-that-trigger-workflows